src/main/java/Bundle_fr.properties

```diff
@@ -0,0 +1,2021 @@
+advanced.search.header.datasets=Ensembles de donn�es
+advanced.search.header.files=Fichiers
+advanced.search.files.name.tip=Le nom donn� au fichier.
+advanced.search.files.description.tip=Un r�sum� d�crivant le fichier et ses variables.
+advanced.search.files.persistentId.tip=L'identifiant permanent du fichier.
+advanced.search.files.persistentId=Identifiant permanent du fichier de donn�es
+advanced.search.files.persistentId.tip=L'identifiant unique permanent pour un fichier de donn�es, lequel peut �tre dans Dataverse un Handle ou un DOI.
+advanced.search.files.fileType=Type de fichier
+advanced.search.files.fileType.tip=L'extension d'un fichier, p.\u00A0ex. CSV, zip, Stata, R, PDF, JPEG, etc.
+advanced.search.files.variableName=Nom de la variable
+advanced.search.files.variableName.tip=Le titre de la colonne pour cette variable dans la table de donn�es.
+advanced.search.files.variableLabel=Libell� de variable
+advanced.search.files.variableLabel.tip=Une courte description de la variable.
+advanced.search.datasets.persistentId.tip=L'identifiant permanent de l'ensemble de donn�es.
+advanced.search.datasets.persistentId=Identifiant permanent de l'ensemble de donn�es
+advanced.search.datasets.persistentId.tip=L'identifiant unique permanent pour un ensemble de donn�es, lequel peut �tre dans Dataverse un Handle ou un DOI.
+
+# search-include-fragment.xhtml
+dataverse.search.advancedSearch=Recherche avanc�e
+dataverse.search.input.watermark=Chercher dans ce dataverse\u2026
+account.search.input.watermark=Chercher ces donn�es\u2026
+dataverse.search.btn.find=Trouver
+dataverse.results.btn.addData=Ajouter des donn�es
+dataverse.results.btn.addData.newDataverse=Nouveau dataverse
+dataverse.results.btn.addData.newDataset=Nouvel ensemble de donn�es
+dataverse.results.dialog.addDataGuest.header=Ajouter des donn�es
+dataverse.results.dialog.addDataGuest.msg=Vous devez <a href="/loginpage.xhtml{0}" title="vous connecter � votre compte Dataverse">vous authentifier</a> pour cr�er un dataverse ou ajouter un ensemble de donn�es.
+dataverse.results.dialog.addDataGuest.msg.signup=Vous devez <a href="{0}" title="Ouvrir un compte Dataverse">ouvrir un compte</a> ou <a href="/loginpage.xhtml{0}" title="vous connecter � votre compte Dataverse">vous connecter</a> pour cr�er un dataverse ou ajouter un ensemble de donn�es.
+dataverse.results.types.dataverses=Dataverses
+dataverse.results.types.datasets=Ensembles de donn�es
+dataverse.results.types.files=Fichiers
+# Bundle file editors, please note that "dataverse.results.empty.zero" is used in a unit test
+dataverse.results.empty.zero=Aucun dataverse, ensemble de donn�es ou fichier ne correspond � votre recherche. Veuillez effectuer une nouvelle recherche en utilisant d'autres termes ou des termes plus g�n�raux. Vous pouvez �galement consulter le <a href="{0}/{1}/user/find-use-data.html" title="Trouver et utiliser des donn�es - Guide d'utilisation de Dataverse" target="_blank">guide de re
```

Comment:
In the English version, there is a rogue "o" character that is probably a typo and should be removed.

Reply:
Yes. Seems also like a typo to me.

Reply:
I will let you do the correction as I am not comfortable in making changes in the English file. Thank you.